Storage/Rust/RustPlaces.swift

```diff
@@ -607,9 +610,55 @@ extension RustPlaces {
         }
     }
 
-    public func getTopFrecentSiteInfos(limit: Int, thresholdOption: FrecencyThresholdOption) -> Deferred<Maybe<[TopFrecentSiteInfo]>> {
-        return withReader { connection in
+    public func getTopFrecentSiteInfos(limit: Int, thresholdOption: FrecencyThresholdOption) -> Deferred<Maybe<[Site]>> {
+        let deferred: Deferred<Maybe<[TopFrecentSiteInfo]>> = withReader { connection in
             return try connection.getTopFrecentSiteInfos(numItems: Int32(limit), thresholdOption: thresholdOption)
         }
+
+        let ret = Deferred<Maybe<[Site]>>()
```

Comment:
nit, rename to something else than `ret`
```suggestion
        let returnValue = Deferred<Maybe<[Site]>>()
```